<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1356</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid #CCCCCC;
    border-radius: 9px;
    margin-top: 8px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="mainGB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    margin-top: 0px;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="renderGB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>191</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Render</string>
       </property>
       <widget class="QCheckBox" name="controlNet">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Control net</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="curvePoints">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Curve points</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="showInfluence">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Influence</string>
        </property>
       </widget>
      </widget>
      <widget class="QSpinBox" name="subdivSteps">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>330</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="maskLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>220</y>
         <width>151</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Mask</string>
       </property>
      </widget>
      <widget class="QLabel" name="stepsLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>300</y>
         <width>151</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Steps</string>
       </property>
      </widget>
      <widget class="QLabel" name="presetLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>151</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
      <widget class="QComboBox" name="netPresets">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>151</width>
         <height>26</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Pentagon</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Basis</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>G</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="subdivMask">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>151</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>1 4 6 4 1</string>
       </property>
      </widget>
      <widget class="QComboBox" name="curveShader">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>390</y>
         <width>141</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circle</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>440</y>
         <width>181</width>
         <height>261</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Influence for a given &lt;/p&gt;&lt;p&gt;control point is shown if&lt;/p&gt;&lt;p&gt;you select a control point.&lt;br/&gt;&lt;/p&gt;&lt;p&gt;The option to render both &lt;/p&gt;&lt;p&gt;curve points and influence&lt;/p&gt;&lt;p&gt;must also be enabled, &lt;/p&gt;&lt;p&gt;and steps &amp;gt; 0.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>370</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Curvature</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="mainView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header>mainview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>controlNet</tabstop>
  <tabstop>curvePoints</tabstop>
  <tabstop>netPresets</tabstop>
  <tabstop>subdivMask</tabstop>
  <tabstop>subdivSteps</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
